mysql-test/suite/rpl/t/rpl_retry_count.inc

```diff
@@ -0,0 +1,90 @@
+# ==== Purpose ====
+#
+# During shutdown of the master, check Slave_IO_Running status 
+# after reconnecting a specified number of times.
+#
+# ==== Usage ====
+#
+# --let $overflow= 0
+# --let $retry_count= 3
+# --let $expected_Slave_IO_Running_status= Connecting
+# --source rpl_mdev36002.inc
+#
+# Parameters:
+#  $overflow
+#   If overflow equals 1, the DEBUG flag 'set_slave_err_count_near_overflow'
+#   will be activated, and the number of tries made will be initialized to
+#   ULONG_MAX - 2.
+#
+#  $retry_count
+#    The number of reconnections the slave will do during shutdown of the master.
+#
+#  $expected_Slave_IO_Running_status
+#    Whether Slave_IO_Running should be Connecting/No
+#    after reconnecting $retry_count times.
+#
+
+
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+
+# Set MASTER_CONNECT_RETRY as 1 second
+--let $rpl_skip_start_slave= 1
+--source include/master-slave.inc
+
+--connection slave
+CHANGE MASTER TO MASTER_CONNECT_RETRY= 1;
+--source include/start_slave.inc
+
+
+# Set err_count as ULONG_MAX if necessary and starts up sync points
+SET @saved_dbug= @@GLOBAL.debug_dbug;
+
+if ($overflow)
+{
+    SET @@GLOBAL.debug_dbug= '+d,set_slave_err_count_near_overflow';
+}
```

Comment:
I may’ve missed this last time –
We use 2-space indents here.
```suggestion
{
  SET @@GLOBAL.debug_dbug= '+d,set_slave_err_count_near_overflow';
}
```
Same for the `while` a bit further from here.